File: Clases2023-24/Clase21_NotificacionesFinal/app/src/main/java/com/example/notificaciones2/MainActivity2.java
package com.example.notificaciones2;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.notificaciones2.databinding.ActivityMain2Binding;

public class MainActivity2 extends AppCompatActivity {

    ActivityMain2Binding binding;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMain2Binding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());


        binding.bVolver.setOnClickListener(
                (boton) -> {
                    finish();
                });
    }
}

File: Clases2023-24/Clase19-DudasFinal/PrietoNoel/app/src/main/java/com/example/prietonoel/MainActivity2.kt
package com.example.prietonoel

import android.os.Bundle
import com.example.prietonoel.databinding.ActivityMain2Binding

class MainActivity2 : ActivityMenu() {

    lateinit var binding: ActivityMain2Binding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMain2Binding.inflate(layoutInflater)
        setContentView(binding.root)

        //la barra de tareas
        setSupportActionBar(binding.miToolbar.toolbar)
        supportActionBar?.title = "Fin App"
        supportActionBar?.setDisplayHomeAsUpEnabled(true)
    }
}